authbridge/authlib/plugins/opa/README.md

```
@@ -0,0 +1,467 @@
+}
+
+allow := {"allow": false, "reason": "task contains sensitive keyword"} if {
+    contains_sensitive_keywords
+}
+```
+
+### Multi-path bundle example
+
+A single bundle can contain rules for multiple decision paths:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)